src/api-reference/common/attendees/v4.attendees.markdown

```diff
@@ -218,6 +219,7 @@ https://{datacenterUri}/v4/attendees?id={id}
 |---|---|---|---|
 `id`|`string`|-| **Required** The attendee ID. Up to 10 comma-separated attendee IDs may be specified.|
 `userId` |`string`|`uuid`|The unique identifier of the SAP Concur user that owns the attendee. Use [Identity v4.1 API](/api-reference/profile/v4.1.identity.html) to retrieve the `userId`. The user who is performing this API request must have the Web Services Administrator user role (Professional) or Can Administer permission (Standard) to use this parameter.|
+`isDeleted`|`boolean`|`true / false`| When `true`, this permits retrieval of a deleted attendee. By default, this value is `false`.|
```

Comment:
will be done in https://jira.concur.com/browse/ATTEND-1587

Reply:
Confirmed with Roger that we should still go ahead and publish as happy path works just fine. We will address in next SU release